### PR TITLE
[LL-7685] Update swap fees wording

### PR DESCRIPTION
**Pull request by ghost**
## 🦒 Context (issues, jira)

https://ledgerhq.atlassian.net/browse/LL-7685

## 💻  Description / Demo (image or video)

Replace "fees" by "network fees"

<details>

<summary> 🖥️  📸   </summary>

</details>

## 🖤  Expectations to reach

PR must pass CI, merge develop if conflicts, do not force push. Thanks!

- **on QA**: at least one of these two checkboxes must be checked:
  - [x] a specific test planned is defined on Jira
  - [x] this PR is covered by automatic UI test
- **on delivery**: at least one of these two checkboxes must be checked: 
  - [ ] Option 1: **no impact**: The changes of this PR have ZERO impact on the userland (invisible for users)
  - [x] Option 2: **atomic delivery**: the changes is atomic and complete (no partial delivery)

PR must pass CI, merge develop if conflicts, do not force push. Thanks!

### CODE REVIEW

File: src/renderer/families/solana/SendRecipientFields.js [hunk 1]
```diff
@@ -4,7 +4,6 @@ import { Trans, withTranslation } from "react-i18next";
 import MemoValueField from "./MemoValueField";
 import Box from "~/renderer/components/Box";
 import Label from "~/renderer/components/Label";
-import LabelInfoTooltip from "~/renderer/components/LabelInfoTooltip";
```
Comment (achabrolin-ledger) [inline src/renderer/families/solana/SendRecipientFields.js:7]: Needed to fix a lint issue

> /actions-runner/_work/ledger-live-desktop/ledger-live-desktop/src/renderer/families/solana/SendRecipientFields.js
7:8 error 'LabelInfoTooltip' is defined but never used no-unused-vars

cf. https://github.com/LedgerHQ/ledger-live-desktop/pull/4597#issuecomment-1004012161

**Comment by achabrolin-ledger** [comment]
CI fails (as of now) because updated screenshots, will be rebased once code review and QA is done and we are ready to merge (to avoid constant rebase)

**Review by LFBarreto — DISMISSED** [review]

**Review by LFBarreto — APPROVED** [review]

**Pull request merged**